www/profile/index.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+$servername = "localhost";
+$username = "worker";
+$password = "swiffcat";
+$db = "socialmedia";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $db);
+
+// Check connection
+if (!$conn) {
+    die("Connection failed: " . mysqli_connect_error());
+}
+
+$headers = getallheaders();
+if (!array_key_exists("token", $headers)) {
+    header("HTTP/1.1 401 Unauthorized");
+    die("{\"reason\":\"token not included\"}");
+}
+$token = getallheaders()["token"];
+
+// If invalid header token, die
+$stmt = $conn->prepare("SELECT user_id FROM auth WHERE token = ?");
+$stmt->bind_param("s", $a1);
+$a1 = $token;
+$stmt->execute();
+$stmt->store_result();
+$user_id = null;
+$stmt->bind_result($user_id);
+if ($stmt->fetch()) {
+    // user_id is bound
+} else {
+    header("HTTP/1.1 401 Unauthorized");
+    die();
+}
+
+if ($_SERVER['REQUEST_METHOD'] === 'GET') {
+    if (!isset($_GET["id"])) {
+        header("HTTP/1.1 403 Forbidden");
+        die("{\"reason\":\"no user id\"}");
```

Comment:
You may put this in here for your debugging but status codes are enough

Reply:
removed them for now.

Reply:
I might make a debug page with these in there later.